.buildkite/pipeline.yml

```
@@ -9,16 +9,14 @@ steps:
     key: "init_cpu_env"
     command:
       - "julia --project=test -e 'using Pkg; Pkg.develop(path=\".\")'"
-      - "julia --project=test -e 'using Pkg; Pkg.rm(\"CLIMAParameters\")'"
-      - "julia --project=test -e 'using Pkg; Pkg.add(url=\"https://github.com/CliMA/CLIMAParameters.jl.git#orad/refactor-parameters-v2\")'"
+      - "julia --project=test -e 'using Pkg; Pkg.add(PackageSpec(\"CLIMAParameters\", rev=\"orad/refactor-parameters-v2\")'"
```

```suggestion
      - "julia --project=test -e 'using Pkg; Pkg.add(PackageSpec(; name=\"CLIMAParameters\", rev=\"orad/refactor-parameters-v2\")'"
```
If this doesn't work, perhaps try this